exchanges/okx/okx_types.go

```diff
@@ -614,13 +710,39 @@ type OrderData struct {
 	SMessage      string `json:"sMsg,omitempty"`
 }
 
+// GetSCode returns a status code value
+func (a *OrderData) GetSCode() string { return a.SCode }
+
+// GetSMsg returns a status message value
+func (a *OrderData) GetSMsg() string { return a.SMessage }
```

Comment:
We could probably RM these methods

Reply:
How? & Why?

Reply:
I might be mistaken, but the data doesn't look like it escapes scope. Have you tested the deployment and response? Also can you please add coverage across all functions that use `handleIncomingData`